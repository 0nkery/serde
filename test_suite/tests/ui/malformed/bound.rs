#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
struct S {
    #[serde(bound(unknown))]
    x: (),
}

fn main() {}
