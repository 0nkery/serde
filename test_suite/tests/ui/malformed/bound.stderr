error: malformed bound attribute, expected `bound(serialize = ..., deserialize = ...)`
 --> $DIR/bound.rs:6:19
  |
6 |     #[serde(bound(unknown))]
  |                   ^^^^^^^

error: aborting due to previous error

